api/handlers.go

```diff
@@ -113,3 +116,57 @@ func (api *RecipeAPI) AddAllRecipeHandler(w http.ResponseWriter, req *http.Reque
 		}
 	}
 }
+
+//AddRecipeHandler - Add a Recipe
+func (api *RecipeAPI) AddRecipeHandler(w http.ResponseWriter, req *http.Request) {
+	ctx := req.Context()
+	vars := mux.Vars(req)
+	recipeID := vars["id"]
+	logD := log.Data{"recipe_id": recipeID}
```

Comment:
These two lines can be removed now the id is not on the URL?

Reply:
Done in commit _"Remove getting id from request to fix unit test"_